sandbox/README.md

```diff
@@ -0,0 +1,13 @@
+# Sandbox
+
+Used for testing the main library.
+
+## Using the sandbox
+
+First, in the root of the project, run:
+
+`scala-cli --power publish local .`
+
+Then in the sandbox directory, run:
+
+`bash run.sh`
```

Comment:
The sandbox doesn't actually work for me, because I get linking errors, almost certainly because I don't have SDL available locally. Suggest you add some instructions somewhere on the best way to do that.